adminguide.rst

```diff
@@ -654,6 +648,69 @@ combinations.
 Troubleshooting
 ---------------
 
+.. _overlay-network:
+
+^^^^^^^^^^^^^^^
+Overlay Netwrok
+^^^^^^^^^^^^^^^
+
+The tree-based overlay network interconnects brokers of the system instance.
+The current status of the overlay subtree at any rank can be shown with:
+
+.. code-block:: console
+
+ $ flux overlay status -r RANK
+
+The possible status values are:
+
+.. list-table:: Subtree Status Values
```

Comment:
This table didn't show up in the readthedocs preview generated for the PR. See https://flux-framework--100.org.readthedocs.build/en/100/adminguide.html#overlay-netwrok

Reply:
Yeah, I went ahead and replace the table with the kind of formatting we used for events in [RFC 20](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_20.html#execution).  Hopeful that will work better.